examples/resources/konnect_api/konnect_api/konnect_api/resource.tf

```diff
@@ -0,0 +1,16 @@
+resource "konnect_api" "my_api" {
+  provider = konnect-beta
+  provider = konnect-beta
+  provider = konnect-beta
```

Comment:
some script is definitely broken :|

Reply:
@mheap I'm going to ignore these for now because they're not the core of this PR  but there's probably something wrong with some script that generates these

Reply:
Aha! That's why I removed the examples directory 🤦 .

Give me a couple of hours to figure this out